nemo/collections/llm/recipes/tp_overlap_configs/userbuffers.py

```diff
@@ -20,6 +20,8 @@ class PipelineOverlapCfg(TPOverlapCfg):
 class RingExchangeOverlapCfg(TPOverlapCfg):
     aggregate: bool = False
     method: str = 'ring_exchange'
+    num_sm: int = 1
+    set_sm_margin: bool = False
```

Comment:
Added this according to config in [NeMo-Framework-Launcher](https://github.com/NVIDIA/NeMo-Framework-Launcher/blob/main/launcher_scripts/conf/training/tp_overlap/ub_cfg_h100_h12288_tp4_mbs1_seqlen2048.yaml#L57)

Reply:
Added defaults according to [T.E base.by](https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/pytorch/module/base.py#L248)